android/src/main/java/vn/hunghd/flutterdownloader/FlutterDownloaderPlugin.java

```diff
@@ -65,7 +65,8 @@ private FlutterDownloaderPlugin(Context context, BinaryMessenger messenger) {
     @SuppressLint("NewApi")
     public static void registerWith(PluginRegistry.Registrar registrar) {
         final FlutterDownloaderPlugin plugin = new FlutterDownloaderPlugin(registrar.context(), registrar.messenger());
-        registrar.activity().getApplication().registerActivityLifecycleCallbacks(plugin);
+        Application application = (Application) registrar.context().getApplicationContext();
```

Comment:
What if we call to `activeContext()` instead of `context()`?

Reply:
Yeah I think that will be enough, but I don't remember if I tried that out or not.

Reply:
In which line?

Reply:
I mean why don't use `registrar.activeContext()`?

Reply:
I tried but didn't work.

Reply:
OK, Thanks for your contribution